src/Components/Display/Menu/UpMenu.tsx

```diff
@@ -42,11 +43,14 @@ class UpMenu extends React.Component<UpMenuProps & WithThemeProps, UpMenuState>{
     constructor(p, c) {
         super(p, c);
         this.state = {
-            minified: this.props.minified,
+            minified: this.props.minified || false,
```

Comment:
minified: this.props.minified || false => utiliser plutôt les defaultProps
{
minified : false
}